test/e2e/framework/cli_lifecycle_operations.go

```diff
@@ -3,12 +3,31 @@
 
 package framework
 
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/vmware-tanzu/tanzu-plugin-runtime/log"
+)
+
+// NewTanzuCLIPathPrefix Tanzu CLI path in the docker image
+const NewTanzuCLIPathPrefix = "/app/tanzu-cli"
+
+// LegacyTanzuCLIPathPrefix Tanzu CLI path in the docker image
+const LegacyTanzuCLIPathPrefix = "/app/legacy-tanzu-cli/v0.28.1"
+
 // CliOps performs basic cli operations
 type CliOps interface {
 	CliInit() error
 	CliVersion() (string, error)
-	InstallCLI(version string) error
-	UninstallCLI(version string) error
+	UninstallTanzuCLI() error
```

Comment:
Please pass the path as a mandatory input parameter so that we can use same method to uninstall legacy or latest tanzu cli